core/payment/examples/invoice_flow.rs

```diff
@@ -33,13 +34,14 @@ async fn main() -> anyhow::Result<()> {
     log::info!("Creating allocation...");
     let allocation = requestor
         .create_allocation(&NewAllocation {
-            address: None,          // Use default address (i.e. identity)
-            payment_platform: None, // Use default payment platform
+            address: None,                                 // Use default address (i.e. identity)
+            payment_platform: Some("ZK-NGNT".to_string()), // Use default payment platform
```

Comment:
Should be made a parameter. This makes the example unusable with other drivers.

Reply:
Reverted this back to use the default driver, this working now